d2renderers/d2sketch/rough.js

```diff
@@ -1261,7 +1261,8 @@ class B {
       n = [];
     if (!t) return this._drawable("path", n, s);
     const i = (function (t, e) {
-      t = (t || "").replace(/\n/g, " ").replace(/(-\s)/g, "-").replace("/(ss)/g", " ");
+      // prettier-ignore
```

Comment:
let's actually add this to https://github.com/terrastruct/d2/blob/master/.prettierignore
it's not made to be read anyway